packages/@aws-cdk/aws-ec2/lib/vpc-lookup.ts

```diff
@@ -38,4 +38,15 @@ export interface VpcLookupOptions {
    * @default Don't care whether we return the default VPC
    */
   readonly isDefault?: boolean;
+
+  /**
+   * Optional tag whose value is used as the name of the subnet group
```

Comment:
This sentence is technically correct but feels complicated because of the indirection (which everyone will understand without even thinking about it).

How about "Tag for subnet group name"?

Reply:
Done.